lib/src/intl_suggestors/intl_importer.dart

```diff
@@ -124,9 +137,34 @@ _InsertionLocation _insertionLocationForPackageImport(
         trailingNewlineCount: 2);
   }
 
+  imports.forEach((importDirective) {
+    final uriContent = importDirective.uriContent;
```

Comment:
This needs to only check package: URLS that are for *this* package. e.g.

import 'package:path/path.dart';
import '../foo.dart';

In this case it is using relative imports only.

Reply:
This is still checking ALL package imports, not just the ones in this package. And the decision process is that we use package imports if there are ANY package imports. So we will not use relative imports if there are imports from any other packages, which is wrong.